modules/standard/Sort.chpl

```diff
@@ -328,7 +328,8 @@ inline proc chpl_compare(a:?t, b:t, comparator:?rec) {
   // Compare results of comparator.key(a) if is defined by user
   if canResolveMethod(comparator, "key", a) {
     // Use the default comparator to compare the integer keys
-    return defaultComparator.compare(comparator.key(a), comparator.key(b));
+    return (new DefaultComparator()).compare(comparator.key(a),
```

Comment:
A local variable would also probably help with some of them.  But this might argue for figuring out a name for a new module-scoped variable instead?  That'd certainly make the deprecation path less annoying for users, since they'd be able to update from `defaultComparator` to `newNamedVar` instead of `defaultComparator` -> `new DefaultComparator()` -> `new defaultComparator()`

Reply:
I think the reason I noticed it here is because you have to wrap the `new` expression in parentheses. I wouldn't expect users to write code like this often with their own custom comparators, and in most cases users don't need to explicitly pass `new defaultComparator()`, since its the default